packages/launcher/package.json

```diff
@@ -77,7 +78,7 @@
     "postinstall": "electron-builder install-app-deps",
     "electron:dev": "concurrently \"PORT=2000 BROWSER=none yarn start\" \"wait-on http://localhost:2000 && tsc -p electron -w\" \"wait-on http://localhost:2000 && tsc -p electron && electron .\"",
     "electron:dev:win": "yarn electron:copy-icon && concurrently \"set PORT=2000 BROWSER=none && yarn start\" \"wait-on http://localhost:2000 && tsc -p electron -w\" \"wait-on http://localhost:2000 && tsc -p electron && electron .\"",
-    "electron:build": "yarn build && tsc -p electron && electron-builder",
+    "electron:build": "yarn build && tsc -p electron && ./node_modules/.bin/electron-builder --projectDir .",
```

Comment:
What does this change do?

Reply:
"If you don’t have dist npm script in your package.json, call ./node_modules/.bin/electron-builder directly."

https://www.electron.build/multi-platform-build

Allows for the app version to actually be recognized by the electron-builder.